src/components/Input.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classes from './Input.module.css';
+
+const Input = () => (
+  <div className={classes.formControl}>
+    <h2>ADD NEW BOOk</h2>
+    <form className={classes.form}>
+      <input type="text" placeholder="Book title" required />
+      <input type="text" placeholder="Author" required />
+      <select name="category" id="category" required>
+        <option value="" disabled selected>
+          Category
+        </option>
+        <option value="Action">Action</option>
+        <option value="Economy">Economy</option>
+        <option value="Fiction">Science Fiction</option>
+        <option value="Adventure">Adventure</option>
+      </select>
+      <button type="button" className={classes.addBtn}>
```

Comment:
- [ ] Please lets the button type be `submit` not `button` though the button wouldn't add the book yet. This shows that it is a submit button.